e2e/testsuite/relayer.go

```diff
@@ -4,21 +4,30 @@ import (
 	"testing"
 
 	dockerclient "github.com/docker/docker/client"
-	"github.com/strangelove-ventures/ibctest"
-	"github.com/strangelove-ventures/ibctest/ibc"
-	"github.com/strangelove-ventures/ibctest/relayer"
+	"github.com/strangelove-ventures/ibctest/v6"
+	"github.com/strangelove-ventures/ibctest/v6/ibc"
+	"github.com/strangelove-ventures/ibctest/v6/relayer"
 	"go.uber.org/zap"
 
 	"github.com/cosmos/ibc-go/e2e/testconfig"
 )
 
 const (
 	cosmosRelayerRepository = "ghcr.io/cosmos/relayer"
+	cosmosRelayerUser       = "100:1000" // docker run -it --rm --entrypoint echo ghcr.io/cosmos/relayer "$(id -u):$(id -g)"
```

Comment:
See [here](https://github.com/strangelove-ventures/ibctest/blob/main/relayer/rly/cosmos_relayer.go#L107)

Reply:
Maybe this should be a const in ibctest.

Reply:
nice suggestion, opened an issue https://github.com/strangelove-ventures/ibctest/issues/321